From 365ee025f61c00250081f1c99cb4d3bbc32a40d4 Mon Sep 17 00:00:00 2001
From: Igor Grinberg <igrinberg@habana.ai>
Date: Mon, 29 Apr 2019 10:03:08 +0300
Subject: [PATCH] [SW-0] pll: fix delay loop for CPU PLL

Commit:
de7021f ([SW-0] pll: make CPU_CYCELS_PER_USEC per device)
accidentally broke the delay loop for cases the CPU runs on CPU PLL,
by a copy/pasteing one zero less.
Fix this by adding the missing zero.

Fixes: de7021f8732bf8d89c2477932fce7e8cb838d6af
Change-Id: I66412e610ec98ed8c61630d19f474c5c578d0832
Signed-off-by: Igor Grinberg <igrinberg@habana.ai>
---

diff --git a/gaudi/gaudi.c b/gaudi/gaudi.c
index 2884fbc..08e89a1 100644
--- a/gaudi/gaudi.c
+++ b/gaudi/gaudi.c
@@ -56,7 +56,7 @@
 
 u64 hl_pll_get_cpu_cycles_per_usec(void)
 {
-	return 120000000ULL / USEC_PER_SEC;
+	return 1200000000ULL / USEC_PER_SEC;
 }
 
 static struct hl_pll_cfg gaudi_pll_cfg[] = {
diff --git a/goya/goya.c b/goya/goya.c
index 3e9aa58..f9a8838 100644
--- a/goya/goya.c
+++ b/goya/goya.c
@@ -182,7 +182,7 @@
 
 u64 hl_pll_get_cpu_cycles_per_usec(void)
 {
-	return 133000000ULL / USEC_PER_SEC;
+	return 1330000000ULL / USEC_PER_SEC;
 }
 
 static struct hl_pll_cfg goya_pll_cfg[] = {
